jsonnet/thanos-ruler.jsonnet

```diff
@@ -114,6 +128,52 @@ local authorizationRole =
         secret.mixin.metadata.withNamespace(thanosRulerConfig.namespace) +
         secret.mixin.metadata.withLabels({ 'k8s-app': 'thanos-ruler' }),
 
+      // query config which holds http configuration
```

Comment:
query config -> alertmanager config

Reply:
also needed to say alertmanager not thanos querier :D